src/lib/transition_frontier_controller_tests/stubs.ml

```diff
@@ -378,6 +378,25 @@ struct
          (root_breadcrumb |> return |> Quickcheck.Generator.return)
          (gen_breadcrumb ~logger ~accounts_with_secret_keys)
 
+  let add_linear_breadcrumbs ~logger ~size ~accounts_with_secret_keys tf =
+    let root = Transition_frontier.root tf in
+    let bcs =
```

Comment:
Don't abbreviate names. I almost didn't got confused by this name as something else other than a breadcrumb.